ci/test_python.sh

```diff
@@ -42,6 +42,9 @@ export LD_PRELOAD="${CONDA_PREFIX}/lib/libgomp.so.1"
 # RAPIDS_DATASET_ROOT_DIR is used by test scripts
 RAPIDS_DATASET_ROOT_DIR="$(realpath datasets)"
 export RAPIDS_DATASET_ROOT_DIR
+pushd "${RAPIDS_DATASET_ROOT_DIR}"
+./get_test_data.sh --subset
+popd
```

Comment:
I don't know for sure if adding this step is necessary or helpful. After we verify that CI passes on the latest commit, we should try removing this before merging.

Reply:
Testing this in #5072.